src/liblinux/BMGUtils.c

```diff
@@ -28,8 +28,8 @@
 #include <stdlib.h>
 
 #include "BMGUtils.h"
-#include "osal_preproc.h"
-#include "liblinux/BMGImage.h"
+#include "../osal_preproc.h"
```

Comment:
This change is also consistent with other declarations, e.g.
https://github.com/mupen64plus/mupen64plus-video-rice/blob/master/src/liblinux/bmp.c#L32
https://github.com/mupen64plus/mupen64plus-video-rice/blob/master/src/liblinux/BMGImage.h#L29
https://github.com/mupen64plus/mupen64plus-video-rice/blob/master/src/liblinux/BMGDLL.h#L45